frb_example/pure_dart/test/api/pseudo_manual/mirror_twin_sync_sse_test.dart

```diff
@@ -149,6 +149,67 @@ Future<void> main({bool skipRustLibInit = false}) async {
     final output = await testHashmapWithMirroredValueTwinSyncSse();
     expect(output.map, {'key': HashMapValue(inner: 'value')});
   });
+
+  test('mirror_enum_stream_twin_normal', () async {
+    final output = await mirrorEnumStreamTwinSyncSse().toList();
+    expect(
+      output,
+      orderedEquals([
+        ApplicationMode.embedded,
+        ApplicationMode.standalone,
+      ]),
+    );
+  });
+
+  test('mirror_option_enum_stream_twin_normal', () async {
+    final output = await mirrorOptionEnumStreamTwinSyncSse().toList();
+    expect(
+      output,
+      orderedEquals([
+        ApplicationMode.embedded,
+        null,
+        ApplicationMode.standalone,
+      ]),
+    );
+  });
+
+  test('mirror_vec_enum_stream_twin_normal', () async {
+    final output = await mirrorVecEnumStreamTwinSyncSse().toList();
+    expect(output, [
+      orderedEquals([ApplicationMode.embedded]),
+      orderedEquals([ApplicationMode.standalone]),
+    ]);
+  });
+
+  test('mirror_map_enum_stream_twin_normal', () async {
+    final output = await mirrorMapEnumStreamTwinSyncSse().toList();
+    expect(output, [
+      allOf(
+        containsPair(0, ApplicationMode.embedded),
+        containsPair(1, ApplicationMode.standalone),
+      )
+    ]);
+  });
+
+  test('mirror_set_enum_stream_twin_normal', () async {
+    final output = await mirrorSetEnumStreamTwinSyncSse().toList();
+    expect(output, [
+      unorderedEquals([
+        ApplicationMode.embedded,
+        ApplicationMode.standalone,
+      ])
+    ]);
+  });
+
+  test('mirror_array_enum_stream_twin_normal', () async {
+    final output = await mirrorArrayEnumStreamTwinSyncSse().toList();
+    expect(output, [
+      orderedEquals([
```

Comment:
nit: IIRC dart supports nested array comparison, e.g. `expect([[1,2]], [[1,2]])`. Is there any reasons to use orderedEquals?

Reply:
Tbh I just looked up for array matchers and the first thing I found was orderedEquals, so no real reason I suppose 😶

Reply:
I see, no problem! If you want to simplify it I will merge after that, otherwise I will merge now.

Reply:
If it's not gonna be a problem, then let's just go ahead 🚀